docs/using/installing/index.rst

```diff
@@ -168,19 +168,26 @@ Using Amazon Web Services
    * `US West (Northern California) <https://console.aws.amazon.com/ec2/v2/home?region=us-west-1#LaunchInstanceWizard:ami=ami-f8f1c8bd>`_
    * `US West (Oregon) <https://console.aws.amazon.com/ec2/v2/home?region=us-west-2#LaunchInstanceWizard:ami=ami-cc8de6fc>`_
 
-#. Configure the instance
-
-   Complete the configuration wizard; in general the default configuration should suffice.
-   However, we do recommend at least the ``m3.large`` instance size.
-
-   If you wish to customize the instance's security settings make sure to permit SSH access both from the intended client machine (for example, your laptop) and from any other instances on which you plan to install ``clusterhq-flocker-node``.
-
-   .. warning::
-
-      Keep in mind that (quite reasonably) the default security settings firewall off all ports other than SSH.
-      E.g. if you run the tutorial you won't be able to access MongoDB over the Internet, nor will other nodes in the cluster.
-      You can choose to expose these ports but keep in mind the consequences of exposing unsecured services to the Internet.
-      Links between nodes will also use public ports but you can configure the AWS VPC to allow network connections between nodes and disallow them from the Internet.
+#. Configure the instance.
+   Complete the configuration wizard; in general the default configuration should suffice.   
+
+   * Choose instance type. We recommend at least the ``m3.large`` instance size.
+   * Configure instance details. You will need to configure a minimum of 2 instances.
+   * Add storage. It is important to note that the default storage of an AWS image can be too small to store popular Docker images, so we recommend choosing at least 16GB to avoid potential disk space problems.
+   * Tag instance.
+   * Configure security group.
+      
+     * If you wish to customize the instance's security settings, make sure to permit SSH access both from the intended client machine (for example, your laptop) and from any other instances on which you plan to install ``clusterhq-flocker-node``.
```

Comment:
This is... inaccurate and ambiguous.

1. There are two things one might mean by "client" - if you mean "flocker-deploy" then SSH access is *not* needed. If you mean "machine admin is using to configure the new instance" then probably saying something like that is more accurate than "client machine".
2. Other instances only need SSH access if they're using ZFS backend... and ZFS on AWS isn't really a supported configuration. 
3. "When you add a custom TCP rule" doesn't quite make sense. "You should add a custom TCP security rule enabling access to ports 4523-4524" perhaps? This is not a nice-to-have, it's a necessary step.